client/pages/general-project-info.tsx

```diff
@@ -0,0 +1,127 @@
+import { Form, Formik } from 'formik';
+import { Field, Radio, Textarea } from '@components';
+
+export const GeneralProjectInfo: React.FC = () => {
+  return (
+    <Formik initialValues={{}} onSubmit={() => {}}>
+      {({ values }: any) => (
+        <Form className='flex justify-center'>
+          <div className='w-2/4 p-4 gap-y-6 bg-white flex flex-col items-center drop-shadow-sm'>
+            <div className='mb-4 flex items-center flex-col'>
+              <h1 className='text-xl font-medium text-bcBluePrimary'>
+                General Project Information and Description
+              </h1>
+              <h3 className=''>Please proivde detailed information for your application.</h3>
+            </div>
+
+            <Field
+              name='estimatedCost'
+              description='Class B or better (in CAD):'
+              type='text'
+              label='Total estimated cost of project'
+              maxLength={300}
+            ></Field>
+
+            <div className='flex flex-1 w-full justify-start'>
+              <Radio
+                legend='What type of project is being applied for?'
+                name='projectType'
+                description='(Please read carefully and then select the one that best fits the project)'
```

Comment:
Description and legend seem to have some semantic/accessibility overlap

I wonder if adjusting `legend`'s props to allow for a react node and then passing markup would be a better solution to ensure the fieldset behaves as expected with assistive technology?